src/types/index.ts

```diff
@@ -832,6 +832,15 @@ export interface DistributionWithDetails {
   details: DividendDistributionDetails;
 }
 
+export interface PaymentDistribution {
+  blockNumber: BigNumber;
+  eventId: string;
```

Comment:
This is not really relevant since it's always `BenefitClaimed`. I assume the middleware team intended to return the event index instead. For now let's just remove it